.jenkins/pace_cache_generation.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -e
+
+if (( $# < 3 )); then
+    echo "USAGE: .jenkins/pace_cache_generation.sh backend experiment [target_uri]"
+fi
+
+backend=$1
+experiment=$2
+
+# If the target_uri starts with "daint:" then the Jenkins scripts are used
+# Use the target_uri as the last command line argument if passed, else the default daint path
+default_target_uri="daint:/scratch/snx3000/olifu/jenkins/scratch/store_gt_caches/$experiment/${backend//:/_}"
```

Comment:
I would suggest breaking out whether or not to use jenkins.sh to run the jenkins as a developer argument or environment variable, instead of folding it in to the path name. This way you can put a comment like "used to run this workflow locally bypassing jenkins.sh" and a reader can know they don't need to understand it in most cases.

As it is, I was pretty confused first reading about this path argument.